activesupport/test/caching_test.rb

```diff
@@ -809,11 +809,6 @@ def test_clear_without_cache_dir
     @cache.clear
   end
 
-  def test_long_keys
-    @cache.write("a"*10000, 1)
```

Comment:
@rafaelfranca I tried to set `10000` key size in `test_really_long_keys`, but test got failed for `MemoryStore`. That happens because we have added fixed hardcoded entry value to define memory store size and some other tests use that size. https://github.com/mechanicles/rails/blob/eb8cd81859bd9f7df8cc08602200847e41e85f0d/activesupport/test/caching_test.rb#L903.

Please note that we share `test_really_long_keys` for other caching stores too.


But when I manually handled size (for memory store) for that test, all tests of that file got passed successfully.